native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/SyntaxTreeGenerator.java

```diff
@@ -309,6 +313,13 @@ public static SyntaxTree generateSyntaxTreeForServiceSample(ServiceStub serviceS
             imports = imports.add(importForTime);
         }
 
+        if (checkForImportsInServices(methodList, "'any:Any")) {
+            ImportDeclarationNode importForTime = Imports.getImportDeclarationNode(
+                    "ballerina", "protobuf.types.'any"
+            );
+            imports = imports.add(importForTime);
```

Comment:
```suggestion
            imports = imports.add(importForAny);
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/509/commits/a318e3547ec9afcb378121d0c9eb61753559f864